src/pages/Reservation.jsx

```diff
@@ -0,0 +1,20 @@
+import '../css/reservation.css'
+import { Link } from 'react-router-dom';
+import search from '../assets/search.png'
+
+const Reservation = () => {
+    return (
+        <div className='reserve-section'>
+            <h1>Book a vespa Testa-Ride</h1>
+            <hr/>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Necessitatibus, ad.</p>
+            <div className="button-container">
+                <Link  to="#!" className="">London</Link >
+                <Link  to="#!" className="">Reserve</Link >
+            </div>
```

Comment:
This part is needed to be a form field with select option of different city or town name , quantity input field with default value 1 and validate the input with isnan only positive.  Reserve is a submit button.